activesupport/lib/active_support/values/time_zone.rb

```diff
@@ -227,6 +228,7 @@ def [](arg)
         case arg
           when String
           begin
+            ActiveSupport::Deprecation.warn 'ActiveSupport::TimeZone[\'Bern\'] is deprecated. Use ActiveSupport::TimeZone[\'Zurich\'] instead' if arg.eql? 'Bern'
```

Comment:
This looks very suspicious here in the code, I think it would be a lot better if there was a list of deprecated timezones instead. (even if it's only one right now)